zapisy/apps/theses/templates/theses/thesis.html

```diff
@@ -44,6 +44,13 @@ <h1 class="d-inline-block">
                     </a>
                 </div>
             {% endif %}
+            {% if is_staff %}
+                <div class="d-inline-block mb-3 ml-1">
+                    <a class="btn btn-sm badge-primary" href="{% url 'admin:theses_thesis_change' thesis.id %}">
+                        Admin
```

Comment:
"Edytuj w panelu administracyjnym" byłoby może za długie, ale może
```suggestion
                        Administruj
```